mac/KeymanEngine4Mac/KeymanEngine4Mac/KME/OnScreenKeyboard/KeyView.m

```diff
@@ -141,6 +144,26 @@ - (void)setKeyCode:(NSUInteger)keyCode {
         _isSpecialKey = YES;
     else
         _isSpecialKey = NO;
+    
+    if (!_isSpecialKey && !_isModifierKey)
+        _isCharacterKey = YES;
+    else
+        _isCharacterKey = NO;
+    
+    // having changed the keycode, adjust the font size of the key label to optimize for available space
+    [self adjustLabelFontSize];
+}
+
+// set the size of the font relative to the key height, depending on what type of key it is
+// this allows larger more readable character key text
```

Comment:
```suggestion
/**
 * set the size of the font relative to the key height, depending on what type of key it is
 * this allows larger more readable character key text
 */
```

Per our [style guide](https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide#Source-Code-Documentation-Syntax), we are trying to use javadoc-style comments for function headers.